"""
A collection of reward functions.
"""

__author__ = 'kongaloosh'


def load_2(data):
    return data[0]


def temperature_2(data):
    return data[1]


def moving_2(data):
    return data[2]


def poisiton_2(data):
    return data[3]


def angle_2(data):
    return data[4]


def vel_command_2(data):
    return data[5]


def load_3(data):
    return data[6]


def temperature_3(data):
    return data[7]


def moving_3(data):
    return data[8]


def poisiton_3(data):
    return data[9]


def angle_3(data):
    return data[10]


def vel_command_3(data):
    return data[11]


def command(data):
    return data[12]
